browser/src/control/Control.Tabs.js

```diff
@@ -334,6 +334,9 @@ L.Control.Tabs = L.Control.extend({
 		if (part !== this._map._docLayer._selectedPart) {
 			this._setPartIndex(part);
 		}
+
+		if (this._map._docLayer.isAutoFillPopupOpen)
```

Comment:
Maybe we won't need this if we close the popup on tab change.

Reply:
(added a comment on `tab change` situation in prior comments)